site/docs/presto.md

```diff
@@ -17,4 +17,4 @@
 
 # Presto
 
-Iceberg connector is part of Presto SQL's master branch, although support for many features is still under development. Current status of the connector is detailed in the [presto-iceberg README](https://github.com/prestosql/presto/blob/master/presto-iceberg/README.md).
+Iceberg connector is part of Presto SQL's master branch, and feature development is complete.  Connector documentation can be found in [Presto Iceberg Connector](https://github.com/prestosql/presto/blob/master/presto-docs/src/main/sphinx/connector/iceberg.rst).
```

Comment:
We should link to the published documentation and not the source code: https://prestosql.io/docs/current/connector/iceberg.html

Reply:
Good point; changed and force-pushed.